src/pages/components/HistoryItem.qml

```diff
@@ -0,0 +1,106 @@
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+ListItem {
+    id: root
+
+    property string search
+
+    function remove() {
+        view.model.remove(model.index)
+    }
+
+    function openLink() {
+        webView.load(model.url, model.title)
+        pageStack.pop()
+    }
+
+    width: view.width
```

Comment:
Default Theme.itemSizeSmall is too small for two label row list item, especially if user changes system font size to "Huge". -> contentHeight: Math.max(Theme.itemSizeMedium, column.height + 2*Theme.paddingMedium)